pkgs/development/tools/rust/cargo-vet/default.nix

```diff
@@ -11,7 +11,7 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-m+2Rbaa7wtzdUyl8VzrGsxtZPhQMwlrx6okhc4zZNsI=";
```

Comment:
I think you also need to update the `rev`

Reply:
that was embarrassing... fixed it